time_test.go

```diff
@@ -0,0 +1,45 @@
+package mono
+
+import (
+	"encoding/json"
+	"testing"
+	"time"
+)
+
+func TestTime_JsonUnmarshal(t *testing.T) {
+	jsonString := []byte(`
+    {
+        "time": 1583000643
+	}`)
+	expectedData := Transaction{
```

Comment:
I don't think, that test should actually now about `Transaction`, let's do simple `type struct{Time Time}` for testing.